databuilder/models/table_metadata.py

```diff
@@ -160,7 +160,7 @@ def __init__(self,
         self.is_view = is_view
         self.attrs = None
         if isinstance(tags, str):
-            tags = tags.split(',')
+            tags = list(filter(None, tags.split(',')))
```

Comment:
should it filter '' or None? For empty tag, will it be empty string?

Reply:
Filtering of None resolved the issue (tested it locally). I guess it's an optimal way:
https://stackoverflow.com/questions/3845423/remove-empty-strings-from-a-list-of-strings

Yes, empty tag - it is in fact a blank/empty string

Reply:
thx, make sense